README.rst

```diff
@@ -62,13 +67,24 @@ To enable this plugin:
     Scrapy setting to
     ``"twisted.internet.asyncioreactor.AsyncioSelectorReactor"``.
 
+In both cases:
+
 -   Set `your Zyte API key
     <https://docs.zyte.com/zyte-api/usage/general.html#authorization>`_ as
     either the ``ZYTE_API_KEY`` Scrapy setting or as an environment variable of
     the same name.
 
 For example, in the ``settings.py`` file of your Scrapy project:
 
+.. code-block:: python
+
+    ADDONS = {
+        "scrapy_zyte_api.ScrapyZyteAPIAddon": 1,
```

Comment:
What do you think about using `scrapy_zyte_api.Addon` name?

Reply:
It's fine for the use case quoted here but I think it's worse when you want to import it, either for subclassing or to use the class in `ADDONS`.

Reply:
You can always rename on import, and the `ADDONS` setting would be a far more common use case, so I also think the shorter name may be worth it.